resttest.rb

```diff
@@ -360,4 +360,17 @@ def test_set_group_roles
 		}
 	end
 
+	def test_websocket		
```

Comment:
I don't understand why these tests are run here. My original thinking behind writing the HTTP API tests inside ruby were that it would make the tests more succinct, vs writing them in Go.
But I can't think why it would make sense to run the imqsauth unit tests from this script. I know we spoke about it, but I can't remember the reasoning now.
One other point - if one was to do it like this, then you'd want to check the program exit code, instead of scanning stdout. Checking `$?.exitstatus == 0` is the most robust way to determine whether a test suite passed.